### PR TITLE
Timer persists with extension shut

### PR DESCRIPTION
**Pull request by gomes404**
Timer now persists after reloading the extension

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -79,28 +127,7 @@ function deleteTask(id) {
     renderTaskList();
 }
 
-// Update task list from chrome.storage.local
-function updateTaskList() {
-    chrome.storage.local.get('syncTasks', (result) => {
-        tasks = result.syncTasks || [];
-        renderTaskList();
-    });
-}
```
Comment (thatsBruno) [inline script.js:88]: Why was this removed?

**Review by thatsBruno — CHANGES_REQUESTED** [review]

File: script.js (hunk 1 above)
Comment (gomes404) [reply script.js:88]: got moved to L79

**Review by thatsBruno — APPROVED** [review]
